src/main/java/it/gov/pagopa/pu/debtpositions/dto/InstallmentPIIDTO.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.pu.debtpositions.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
+public class InstallmentPIIDTO {
+
+    private String uniqueIdentifierType;
+    private String uniqueIdentifierCode;
+    private String fullName;
+    private String address;
+    private String civic;
+    private String postalCode;
+    private String location;
+    private String province;
+    private String nation;
+    private String email;
```

Comment:
it would be better to understand having inside the InstallmentPIIDTO the Installment fields segregated.
This way, the mapping operation between Installment and InstallmenteNoPii/InstallmentPII would be easier (you will have not to transform this into Person).
```suggestion
    private Person debtor;
```